Functions/BIDS/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -151,8 +161,16 @@
 		if dimASL(4)>=numel(jsonOut.GELabelingDuration)
 			warning('Labeling duration mismatch with GE private field.');
```

Comment:
which private field?

Reply:
clarified that it's the studyPar versus DICOM. See 43d03f7b

Reply:
No sorry that's not what I meant. I meant that you actually print WHICH private field mismatched. E.g.,:

>> warning('studyPar parameter LabelingDuration=1.6 mismatch with DICOM field LabDur=1.8, using 1.8.'); (in the case that the DICOM field is called LabDur)

Again, this makes it easier for us to help users and check for bugs, as you have 1000 potential mismatch warnings, you better specifiy what they are.

Reply:
Sure. Clarifying warnings makes sense. Done as suggested in 337e00c4